docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,71 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Reduce number of requests from coordinator to workers ({issue}`8460`)
+* Improve query performance by sending collected dynamic filters from coordinator to workers. ({issue}`5183`)
+* Improve performance of inequality joins where join condition sides have different type. ({issue}`8500`)
+* Fix query failure when there is a constant predicate on some
+  column `col` (e.g `col=1`), followed by  `ORDER BY col` and `LIMIT`. ({issue}`8535`)
+* Fix SHOW CREATE SCHEMA. ({issue}`8598`)
```

Comment:
```suggestion
* Fix `SHOW CREATE SCHEMA`. ({issue}`8598`)
```

Reply:
BTW, what was fixed? Was it a failure? Correctness issue?

Reply:
Queries failed for connectors which didn't support schema properties IIRC.

Reply:
Failure. per https://github.com/trinodb/trino/issues/8455#issuecomment-883276242, it applies to all connectors except of Hive, Iceberg, Memory, Blackhole

Reply:
Thanks. I'll change it to "Fix SHOW CREATE SCHEMA failure"